util/util.go

```diff
@@ -148,6 +148,12 @@ func ResolveName(name string, firstRegistry string, sc *types.SystemContext, sto
 	return candidates, DefaultTransport, searchRegistriesAreEmpty, nil
 }
 
+// startsWithValidTransport validates the name starts with Buildah supported transport
+// to avoid the corner case image name same as the transport name
+func startsWithValidTransport(name string) bool {
```

Comment:
We might want to make this global at some point.

Reply:
fixed